addons/stock_delivery/wizard/choose_delivery_package.py

```diff
@@ -42,11 +45,11 @@ def _onchange_package_type_weight(self):
             return {'warning': warning_mess}
 
     def action_put_in_pack(self):
-        move_line_ids = self.picking_id._package_move_lines(batch_pack=self.env.context.get("batch_pack"))
-        delivery_package = self.picking_id._put_in_pack(move_line_ids)
+        move_line_ids = self.move_line_ids._package_move_lines()
```

Comment:
I think it would be good idea to add a kwargs in action_put_in_pack in order to pass the weight. Rather than duplicate the method content to add the additional data